src/main/java/codeu/model/data/UserProfile.java

```diff
@@ -0,0 +1,49 @@
+
+
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+
+/** Class representing a User's Profile info. It is shown on a User's Profile Page */
+public class UserProfile {
+  private final UUID id;
+  private final String aboutMe;
+  private final String sentMessages;
+  private final Instant creation;
+
+  /**
+   * Constructs a new UserInfo.
+   *
+   * @param id the ID of this User
+   * @param aboutMe the aboutMe info of this User
+   * @param sentMessages the past messages sent by this User
+   * @param creation the creation time of this User
+   */
+  public UserProfile(UUID id, String aboutMe, String sentMessages, Instant creation) {
```

Comment:
What does the model look like for sentMessages?

Using just a single `String` seems problematic. What if the user has sent a million messages?